lib/system.nim

```diff
@@ -3791,10 +3805,8 @@ template doAssert*(cond: bool, msg = "") =
   # NOTE: `true` is correct here; --excessiveStackTrace:on will control whether
   # or not to output full paths.
   {.line: instantiationInfo(-1, true).}:
-    if not cond:
-      raiseAssert(astToStr(cond) & ' ' &
-                  instantiationInfo(-1, false).fileName & '(' &
```

Comment:
Why don't you just change this to ``true`` and be done with it?

Reply:
I don't understand your comment; that would change semantics

Reply:
simplified this, see updated PR